webapp/src/store/cards.ts

```diff
@@ -168,6 +168,12 @@ export function getCard(cardId: string): (state: RootState) => Card|undefined {
     }
 }
 
+export function isCardEmpty(cardId: string): (state: RootState) => boolean {
+    return (state: RootState): boolean => {
+        return getCards(state)[cardId]?.title === '' && getCards(state)[cardId]?.fields.contentOrder.length === 0 && Object.keys(getCards(state)[cardId]?.fields?.properties).length === 0 && !state.comments.commentsByCard[cardId] && !state.attachments.attachmentsByCard[cardId]
+    }
+}
+
```

Comment:
Suggest using existing selectors to retrieve data vs accessing state directly.

Reply:
I moved the `isCardEmpty` function into a hook and used the selectors there in my latest commit, please let me know if this is okay